stable/snipeit/templates/deployment.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "snipeit.fullname" . }}
+  labels:
+    app: {{ template "snipeit.name" . }}
+    chart: {{ template "snipeit.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  revisionHistoryLimit: {{ .Values.revisionHistoryLimit }}
+  selector:
+    matchLabels:
+      component: app
```

Comment:
The component label should also be on the deployment itself. But, since there is only one component, do you need it at all?

Reply:
Hi,
I updated the labels. I had some DCO-signing problems after my vacation, that's why commit history looks a bit messy - but it works now. 
Hope everything is alright now

thanks